osu.Game.Rulesets.Catch/Objects/JuiceStream.cs

```diff
@@ -142,13 +140,8 @@ public SliderPath Path
             set
             {
                 path.ControlPoints.Clear();
-                path.ExpectedDistance.Value = null;
-
-                if (value != null)
```

Comment:
This one is really weird but I can't see where it's coming from, so... Seems safe-ish to remove.

Reply:
It can break deserialization of a bad JSON (that isn't produced by the current JSON serialization).
It shouldn't break anything because JSON serialization is currently only used internally.